lib/vehicle_factory.rb

```diff
@@ -0,0 +1,14 @@
+class VehicleFactory
+  def initialize
+  end
+  
+  def create_vehicles(api_data)
+    @factory_vehicles = []
+    api_data.select do |data|
```

Comment:
`select` is an enumerable best used for only keeping elements when a certain condition about those elements evaluates to true. Since lines 8-10 will always return truthy values (meaning: they don't contain any conditional logic or filter anything out), I might reach for `.map` here instead.